lucien-ios/APIClient/LucienRequest.swift

```diff
@@ -74,12 +97,34 @@ enum LucienRequest: Request {
             return nil
         case .hasCollection:
             return nil
+        case .createPhotoURL:
+            return nil
+        case .addComicBook(let comicTitle,
+                           let storyTitle,
+                           let volume,
+                           let issueNumber,
+                           let publisher,
+                           let releaseYear,
+                           let comicPhotoURL,
+                           let returnDate,
+                           let condition,
+                           let genre):
+            return ["comic": ["comic_title": comicTitle,
```

Comment:
Instead of creating this on the fly, it should be a property on `Comic` and I suspect we can use `Codable` to generate this for us